### DISCUSSION TITLE
Preset machine configurations are grayed out

### DISCUSSION THREAD

**Discussion by E1TechPro** [artisan-roaster-scope/artisan#1030] [General]
I just downloaded the most recent version of Artisan (v2.8.1).  I am trying to configure the software to support my Hottop 2K+, but the machine name is grayed out.  As a matter of fact, when I click on the Config menu, select Machines, I see a list of manufacturers, but none of their machines are "selectable" - they are all grayed out.  Any help is appreciated!
<img width="1280" alt="Screen Shot 2022-12-03 at 1 28 37 PM" src="https://user-images.githubusercontent.com/62855855/205456287-d429fd7b-23ca-4dfb-ab58-128e54a0b70c.png">

**Comment by MAKOMO** [comment]
Your Artisan is running in ArtisanViewer mode (shown in the main windows title bar, but hidden in your screenshot). There must be another Artisan process running on this machine. The second Artisan instance always starts in ArtisanViewer mode which is not able to communicate to devices.

Maybe this other Artisan process is hanging. Kill it or just restart our mac to get rid of it.

**Reply by E1TechPro** [reply]
I rebooted my MacBook and that resolved this issue.  Thank you!

So, now that I am able to select the Hottop 2k+ preset configuration, I did the following:

- Connected to the Hottop using the USB cable; placed Hottop in "Auto" mode
- In Artisan, changed the Port setting in Config->Port to FT230X Basic UART
- Verified that the Mac saw the device by inspecting the System Report->USB, and verified that the roaster was connected
- Right clicked on the timer to turn Super on
- Clicked on the blue "Control" button, yet, the roaster is not responding

Any ideas on next steps?

Thank you,
Rich

**Reply by E1TechPro** [reply]
So, after playing with this for another hour, I was able to get it to work.  It must have been a sequence of steps that I was missing.  Thank you for a speedy respond to my initial post -- will make a donation as gratitude for the great product, and great service.
..

**Comment by E1TechPro** [comment]
I rebooted my MacBook and that resolved this issue.  So, now that I am able to select the Hottop 2k+ preset configuration, I did the following:
- Connected to the Hottop using the USB cable; placed Hottop in "Auto" mode- In Artisan, changed the Port setting in Config->Port to FT230X Basic UART- Verified that the Mac saw the device by inspecting the System Report->USB, and verified that the roaster was connected
- Right clicked on the timer to turn Super on- Clicked on the blue "Control" button, yet, the roaster is not responding
Any ideas on next steps?
Thank you,
Rich
    On Saturday, December 3, 2022, 01:55:54 PM EST, Marko Luther ***@***.***> wrote:  
 
 


Your Artisan is running in ArtisanViewer mode (shown in the main windows title bar, but hidden in your screenshot). There must be another Artisan process running on this machine. The second Artisan instance always starts in ArtisanViewer mode which is not able to communicate to devices.

Maybe this other Artisan process is hanging. Kill it or just restart our mac to get rid of it.
